landingpage2.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Swipe</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link rel="stylesheet" href="./styles2.css" />
+  </head>
+  <body>
+    <section class="header-seccion_1">
+      <div class="background-header"></div>
+      <header>
+        <nav class="menu">
+          <img class="swipe" src="./assets/logo-white.svg" alt="logo" />
```

Comment:
podrías borrar el "/" antes de cada assets y probar si funciona?

Algo así: 
```suggestion
          <img class="swipe" src="assets/logo-white.svg" alt="logo" />
```

Reply:
hola dani, se corrige el código quitando el punto(.) y la barra inclinada (/). en las zonas donde dejaste tu obsevación.